Sources/Java2Swift/JavaToSwift.swift

```diff
@@ -228,12 +228,27 @@ struct JavaToSwift: ParsableCommand {
 
       // Note that we will be translating this Java class, so it is a known class.
       translator.translatedClasses[javaClassName] = (translatedSwiftName, nil, true)
+
+      var classes: [JavaClass<JavaObject>?] = javaClass.getClasses()
+
+      // Go through all subclasses to find all of the classes to translate
+      while let internalClass = classes.popLast() {
+        if let internalClass {
+          let (javaName, swiftName) = names(from: internalClass.getName())
+          // If we have already been through this class, don't go through it again
+          guard translator.translatedClasses[javaName] == nil else { continue }
+          let currentClassName = swiftName
+          let currentSanitizedClassName = currentClassName.replacing("$", with: ".")
+          classes.append(contentsOf: internalClass.getClasses())
+          translator.translatedClasses[javaName] = (currentSanitizedClassName, nil, true)
+        }
+      }
     }
 
     // Translate all of the Java classes into Swift classes.
     for javaClass in javaClasses {
       translator.startNewFile()
-      let swiftClassDecls = translator.translateClass(javaClass)
+      let swiftClassDecls = try! translator.translateClass(javaClass)
```

Comment:
Shouldn't the be a `try` rather than a `try!`?

```suggestion
      let swiftClassDecls = try translator.translateClass(javaClass)
```

Reply:
I was just following what was there before (there was a try! inside, I just moved it out). I'm fine making this a try